--- ./src/glog/stl_logging.h.in.orig	2013-10-31 17:27:59.000000000 +0000
+++ ./src/glog/stl_logging.h.in	2013-10-31 17:29:52.000000000 +0000
@@ -53,6 +53,8 @@
 #ifdef __GNUC__
 # include <ext/hash_set>
 # include <ext/hash_map>
+#endif
+#ifdef GLOG_STL_LOGGING_FOR_EXT_SLIST
 # include <ext/slist>
 #endif
 
@@ -80,7 +82,7 @@
 OUTPUT_TWO_ARG_CONTAINER(std::vector)
 OUTPUT_TWO_ARG_CONTAINER(std::deque)
 OUTPUT_TWO_ARG_CONTAINER(std::list)
-#ifdef __GNUC__
+#ifdef GLOG_STL_LOGGING_FOR_EXT_SLIST
 OUTPUT_TWO_ARG_CONTAINER(__gnu_cxx::slist)
 #endif
 
